### PR TITLE
update

### PR DESCRIPTION
**Pull request by choper725**

### CODE REVIEW

**Review by oblomov-dev — COMMENTED** [review]
Is ty_chart needed?

**Comment by choper725** [comment]
this type is used when you build a chart thats why the error not used in the cc class

File: src/02/z2ui5_cl_cc_chartjs.clas.abap [hunk 1]
```diff
@@ -492,15 +502,15 @@ CLASS z2ui5_cl_cc_chartjs DEFINITION
     TYPES:
       BEGIN OF ty_chart,
```
Comment (larshp) [inline src/02/z2ui5_cl_cc_chartjs.clas.abap:503]:
```suggestion
      BEGIN OF ty_chart ##NEEDED,
```

**Comment by oblomov-dev** [comment]
@choper725 next time you can fix the unused issue with the ##NEEDED pragma:
image

Thank you as always for your changes, just fyi, i get an ID error in demo 148:
image

File: src/02/z2ui5_cl_cc_chartjs.clas.abap (hunk 1 above)
Comment (oblomov-dev) [reply src/02/z2ui5_cl_cc_chartjs.clas.abap]: thanks!

**Pull request merged**